src/compo/airnow2ioda_nc.py

```diff
@@ -1,7 +1,14 @@
 #!/usr/bin/env python3
 
+#
+# (C) Copyright 2024 UCAR
+#
+# This software is licensed under the terms of the Apache Licence Version 2.0
+# which can be obtained at http://www.apache.org/licenses/LICENSE-2.0.
+#
 # Read airnow text data file and convert to IODA netcdf
-import os
+
+import os, sys
```

Comment:
```suggestion
import os
```

i do not see this being used